tests/test_csc.py

```diff
@@ -167,8 +176,22 @@ async def doit():
 
         asyncio.get_event_loop().run_until_complete(doit())
 
+# FIXME: WHY DOESN"T THIS WORK?? It needs to be inside a given test??
```

Comment:
Any reason for this to be here? can you remove it?

Reply:
This was meant as a question to you :)

Reply:
If you define the method here it should be a class method. That means the first argument must be `self`, e.g.;

```
async def publish_mountEncoders(self, harness, azimuth, elevation, ntimes=5):
```

But, it doesn't seem like this is using anything from the class so you would probably want to define it as a `@staticmethod`;

```
    @staticmethod
    async def publish_mountEncoders(harness, azimuth, elevation, ntimes=5):
```

But still, not sure this is the best approach either. I would just define it inside the test method or outside the scope of the class, as a regular top level method.

Reply:
ya, declaring inside the class was the issue I think... works now. Thanks